src/test/java/org/prebid/server/bidder/rubicon/RubiconBidderTest.java

```diff
@@ -332,9 +334,86 @@ public void makeHttpRequestsShouldFillVideoExt() {
                 .extracting(Imp::getVideo).doesNotContainNull()
                 .extracting(Video::getExt).doesNotContainNull()
                 .extracting(ext -> mapper.treeToValue(ext, RubiconVideoExt.class))
-                .containsOnly(RubiconVideoExt.of(5, 10, RubiconVideoExtRp.of(14)));
+                .containsOnly(RubiconVideoExt.of(5, 10, RubiconVideoExtRp.of(14), null));
     }
 
+    @Test
+    public void makeHttpRequestsShouldFillVideoExtAndAddVideoType() {
+
```

Comment:
Please add a `//given` inline commentary for a block of test where you declare variables or set up stubs.

Check other missing cases. Thanks!